Modify rules S1774,S3358: Add exception for C++11 constexpr functions (#1009)
SonarSource/rspec: rules/S1774/cfamily/rule.adoc

include::../rule.adoc[]

== Exceptions

For {cpp}11 mode only, the issue is not raised for ternary operators used inside `constexpr` functions. 

> In {cpp}11 such functions are limited to single statement, that is `return` so use of ternary operator

Reviewer: ```suggestion
In {cpp}11 such functions are limited to a single return statement, so the use of a ternary operator
```

Author: This does not convey wording. Being limited to `single return statement` can be read as having multiple statements, but only one `return`. But this function can have only one statement, that is also `return`.

Author: Looks good.
